perl-Bootloader-testsuite/tests/test_interface/backup/fake_root1/boot/extlinux/extlinux.conf

```diff
@@ -0,0 +1,25 @@
+TIMEOUT 1000
+ONTIMEOUT localboot
+DEFAULT default
+MENU TITLE Boot menu
+PROMPT 0
+
+LABEL default
+         MENU LABEL Default
+         LINUX /boot/zImage
+         FDTDIR /boot/dtb/
+         FDT exynos5250-snow.dtb
```

Comment:
if FDT is specified, it is the whole path, fdtdir has no effect.
http://code.metager.de/source/xref/denx/u-boot/doc/README.pxe#158

fdtdir is especially useful for the RPi, the fdt filename is generated by u-boot from OTP (one-time-programmable) memory in the SoC.
http://git.denx.de/?p=u-boot.git;a=blob;f=board/raspberrypi/rpi/rpi.c#l77

In fact, fdtdir should be used in general.